.github/container/pip-finalize.sh

```diff
@@ -10,14 +10,23 @@ pushd /opt/pip-tools.d
 # twice
 pip-compile -o requirements.pre $(ls requirements-*.in)
 
-IFS=$'\n'
-for line in $(cat requirements.pre | egrep '^[^#].+ @ git\+' || true); do
-  # VCS installs are of the form "PACKAGE @ git+..."
-  PACKAGE=$(echo "$line" | awk '{print $1}')
-  ref=$(yq e ".${PACKAGE}.latest_verified_commit" ${MANIFEST_FILE})
-  echo "${line}@${ref}"
+# Find the VCS installs, which are of the form
+# PACKAGE @ git+GIT_REPO_URL
+for line in $(sed -n -e 's/^\([^#].*\) @ git+\(.*\)$/\1=\2/p' requirements.pre); do
```

Comment:
Why did the regex expression change? I do understand the logic afterwards (19:27) but just not how records in `requirements.pre` differ such that the regrex matcher needs to be changed.